rskj-core/src/test/java/co/rsk/mine/MinerServerTest.java

```diff
@@ -923,4 +692,73 @@ private void findNonce(MinerWork work, BtcBlock bitcoinMergedMiningBlock) {
             }
         }
     }
+
+    private MinerServerImpl makeMinerServer(EthereumImpl ethereumImpl, BlockUnclesValidationRule unclesValidationRule,
```

Comment:
Beatiful refactorings in the whole file! Only a suggestion: `makeMinerServer` can be called `makeMinerServerMock`, I think it easier to understand what's happening while reading the method invocation. As the other suggestions, is only an opinion, feel free to dismiss it if you don't like it.

Reply:
Excellent Idea! we can add it to the readme, and establish some procedure like "When doing or modifying some test on an existent class, we should try to update as many names as possible to comply with the naming convention" or something like that.